src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,81 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+
+    @OneToOne
+    @JoinColumn(name = "cinema_hall_id")
+    private CinemaHall cinemaHall;
```

Comment:
And also here one cinema hall can have many movie sessions
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "cinema_hall_id")
    private CinemaHall cinemaHall;
```